### PR TITLE
Cleancss fix

### PR DESCRIPTION
**Pull request by matteofigus**
With the recent upgrade of oc-template-jade (#497) using a patched jade version with security fixes, the `uglify-css` filter has gone in favour of newest `clean-css`. This means that components using this filter in production would break.

This introduces support for the old filter, allowing a clean migration to the new filter (by publishing a new component that uses `clean-css` instead of `uglify-css`.

In order to test it works, I added a component that has all the filters (jade-filters) which I test doesn't generate any compilation error during rendering.

### CODE REVIEW

File: test/fixtures/components/jade-filters/.npmignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+_package
+package.tar.gz
+node_modules
```
Comment (mattiaerre) [inline test/fixtures/components/jade-filters/.npmignore]: newline

File: test/fixtures/components/jade-filters/files/file.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var sum = function(x, y){
```
Comment (mattiaerre) [inline test/fixtures/components/jade-filters/files/file.js]: const

File: test/fixtures/components/jade-filters/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "oc": {
+        "files": {
+            "data": "server.js",
+            "template": {
+                "src": "template.jade",
+                "type": "jade"
+            }
+        }
+    }
+}
```
Comment (mattiaerre) [inline test/fixtures/components/jade-filters/package.json]: newline

File: test/fixtures/components/jade-filters/files/file.js (hunk 2 above)
Comment (matteofigus) [inline test/fixtures/components/jade-filters/files/file.js]: client-side js

**Comment by matteofigus** [comment]
@mattiaerre any time jade/pug encounters a filter, it tries to resolve the filter by requiring `jstransformer-${filterName}`. By renaming the `uglify-css` to `clean-css` they basically broke retro-compatibility. As you can see from the package.json, I created a filter that adds the uglify-css alias back: https://github.com/matteofigus/jstransformer-uglify-css/blob/master/index.js

File: test/fixtures/components/jade-filters/.npmignore (hunk 1 above)
Comment (matteofigus) [inline test/fixtures/components/jade-filters/.npmignore]: done

File: test/fixtures/components/jade-filters/package.json (hunk 3 above)
Comment (matteofigus) [inline test/fixtures/components/jade-filters/package.json]: done

**Comment by matteofigus** [comment]
This is the code that the jade-legacy removed in favour of migrating to jstransformers, which basically broke the uglify-css filter: https://github.com/jnordberg/jade-legacy/commit/0ec5fc6ddacbe40f41bf78a8b694e394027796a5

File: test/fixtures/components/jade-filters/files/file.js (hunk 2 above)
Comment (mattiaerre) [inline test/fixtures/components/jade-filters/files/file.js]: kk

**Comment by mattiaerre** [comment]
@matteofigus this LGTM 👍

**Comment by matteofigus** [comment]
Thanks for your quick help @mattiaerre

**Comment by mattiaerre** [comment]
@matteofigus let me know if there is anything else I can do in order to help you w/ testing this new functionality once it has been released

**Pull request merged**